.travis.yml

```
@@ -41,6 +41,7 @@ install:
 script:
   - flake8 --max-line-length=110
   - python check_keys.py
+  - python -m doctest README.rst
```

```suggestion
  - python -m doctest README.rst -v
```

I propose to add verbosity here to 
- better understanding, what's going on on this stage
- make sure than a fresh "piece of code" was interpreted as docstring (to avoid the situation that happened yesterday).